src/front-end/src/components/MenteeForm.js

```diff
@@ -0,0 +1,58 @@
+/* eslint-disable */
+import React, { Component } from 'react';
+import { Form, Button, Input, DatePicker, Tooltip, Select } from 'antd';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { InfoCircleOutlined, QuestionCircleOutlined } from '@ant-design/icons';

```

Comment:
Why not remove the extraneous dependencies imported here? Then we should be able to get rid of this linting ignore.

Reply:
![image](https://user-images.githubusercontent.com/71383839/99034765-c9d98380-2532-11eb-863a-73e83c0ba4b3.png)
if I remove it, the error appears. And I'm unable to run the given command.

Reply:
I checkout your code, it looks like you managed to fix this?